validator/src/main/resources/expected-data-template/otlpExpectedLog.mustache

```diff
@@ -0,0 +1,10 @@
+{
+    "body": "Executing outgoing-http-call",
+    "severity_number": "{{severity_number}}",
+    "severity_text": "INFO",
+    "trace_id": "{{trace_id}}",
+    "span_id": "{{span_id}}",
```

Comment:
Where do these values get populated from? Are they present in the context?

Reply:
I believe they are being populated from the actual log message. The goal wasn't to test that the values matched up, rather, to confirm that there were these fields present in the actual log message.